### ISSUE TITLE
RootUrl not working?

### ISSUE THREAD

**Issue by SjaaLom** [Legedric/ptmagic#517]
Hi Legedric,

I was sorry to see that you closed my previous question as being a "unix specific issue" (which I don't believe it to be). But I think there might still be an issue here.

Can you please take a look at the Monitor.RootUrl property from the settings.general.json file. When I change it to something other than "/" I am not able to open PTM anymore in the browser.

I tested this on Windows and Linux (and could even test it on Mac later today, if you would like)

**Comment by Legedric** [comment]
Opening another issue won't change anything.

The setting RootUrl was implemented to help people using ngingx reverse proxy as it was brought to our attention in #239 

We personally do not use it and I will not set this up on my machine just to test a proxy configuration on Windows.. So far I did not get any feedback from someone using reverse proxy that it doesn't work.
But as I won't setup a Linux machine for this and dig myself into that topic you are on your own.

Sorry for not being any help for your issue here but that's how it is as my/our time is extremely limited and when it comes down to that kind of specialities we have to simply prioritize what topics we want to spend our time on.
